test/gerkin/tests/features/data_exist.feature

```diff
@@ -0,0 +1,20 @@
+Feature: API endopont allowing users to check if a record already exists in the Digital Registries database.
+  Request endpoint: POST /data/{registryname}/{versionnumber}/exists
+
+  Background:
+    Given The user wants to check if a record with the first name "John Helmut" already exists in the Digital Registries database
+
+  Scenario: The user receives a message that the record already exists in the Digital Registries database
+    When The user triggers an action to check if the record already exists in the database
+    And The request with a valid payload is sent
```

Comment:
Scenarios in this feature are missing a little bit more details. There's no distinction between them if you look only at the 'when' and 'and' cases but the difference should be that the data that the user is looking for is either missing or existing in the database.

Add additional information:
`And The searched record already exists`
or
`And The searched record doesn't exist`